apps/worker/src/app/workflow/services/workflow.worker.ts

```diff
@@ -44,7 +45,7 @@ export class WorkflowWorker extends WorkflowWorkerService implements INovuWorker
 
             storage.run(new Store(PinoLogger.root), () => {
               _this.triggerEventUsecase
-                .execute(data)
+                .execute(data as TriggerEventCommand)
```

Comment:
We should create a command here, but before and solve this issue of complex command interface https://github.com/novuhq/novu/pull/4843/files#r1392301028